types/ghidra.js/.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
+    "rules": {
+        "@definitelytyped/no-const-enum": "off",
+        "@definitelytyped/dt-header": "off",
+        "@definitelytyped/no-unnecessary-generics": "off",
+        "@definitelytyped/strict-export-declare-modifiers": "off",
+        "@typescript-eslint/naming-convention": "off"
```

Comment:
Please don't disable any lint rules for new types. The publish won't even work without the proper DT header, and it has syntax for non-npm package names that can squelch the naming convention error, too.

Const enums are generally frowned upon in DT - there're a performance optimization for within your own project, and aren't really safe to use across project boundaries (as they will be version-locked at compile time via the inling process!).

Unnecessary generics just mask a location that should just be an `unknown` and a cast.

`declare` modifier strictness is mostly just a style rule, but new types are supposed to conform to it~

Reply:
I'm sorry, I just copypasted this file from another module :) WIll remote it soon

Reply:
Removed all local rules from `.eslintrc.json` @weswigham